src/generate.jl

```diff
@@ -5,6 +5,7 @@ function generate(ctx::Context, path::String; kwargs...)
     Context!(ctx; kwargs...)
     ctx.preview && preview_info()
     dir, pkg = dirname(path), basename(path)
+    Base.isidentifier(pkg) || pkgerror("$pkg is not a legal Julia identifier")
```

Comment:
```suggestion
    Base.isidentifier(pkg) || pkgerror("$(repr(pkg)) is not a valid package name")
```

Reply:
I think package might be best as usual case, but it might be more proper to say project name in the project/package/application lingo.